types.d.ts

```diff
@@ -15,33 +15,59 @@ export interface BlogMiddleware {
 export type DateStyle = "full" | "long" | "medium" | "short";
 
 export interface BlogSettings {
+  // The blog title
```

Comment:
I believe you need to use `/** */` for these to properly show up in the doc generator, `//` are considered "internal comments"